packages/playground-server/test/api.spec.ts

```diff
@@ -12,18 +15,107 @@ const client = axios.create({
 });
 
 describe("playground-server", () => {
-  beforeEach(async () => {
-    server = await api.listen(9001);
+  beforeEach(done => {
+    server = api.listen(9001, done);
   });
-  it("exposes /api/hello", async () => {
-    const response = await client.get("/hello", {
-      params: {
-        name: "Alice"
-      }
-    });
-    expect(response.data.hello).toEqual("Alice");
+  afterEach(done => {
+    server.close(done);
   });
-  afterEach(() => {
-    server.close();
+  describe("/api/create-account", () => {
+    it("fails when username is not passed to the request", () => {
+      client.post("/create-account").catch(({ response }) => {
+        expect(response.status).toEqual(400);
+        expect(response.data).toEqual({
+          ok: false,
+          error: {
+            status: 400,
+            errorCode: ErrorCode.UsernameRequired
+          }
+        });
+      });
+    });
+    it("fails when email is not passed to the request", () => {
+      client
+        .post("/create-account", {
+          username: "alice"
+        })
+        .catch(({ response }) => {
+          expect(response.status).toEqual(400);
+          expect(response.data).toEqual({
+            ok: false,
+            error: {
+              status: 400,
+              errorCode: ErrorCode.EmailRequired
+            }
+          });
+        });
+    });
+    it("fails when address is not passed to the request", async () => {
+      client
+        .post("/create-account", {
+          username: "alice",
+          email: "alice@wonder.land"
+        })
+        .catch(({ response }) => {
+          expect(response.status).toEqual(400);
+          expect(response.data).toEqual({
+            ok: false,
+            error: {
+              status: 400,
+              errorCode: ErrorCode.AddressRequired
+            }
+          });
+        });
+    });
+    it("fails when signature is not passed to the request", async () => {
+      client
+        .post("/create-account", {
+          username: "alice",
+          email: "alice@wonder.land",
+          address: "0x0f693cc956df59dec24bb1c605ac94cadce6014d"
+        })
+        .catch(({ response }) => {
+          expect(response.status).toEqual(400);
+          expect(response.data).toEqual({
+            ok: false,
+            error: {
+              status: 400,
+              errorCode: ErrorCode.SignatureRequired
+            }
+          });
+        });
+    });
+    it("fails when an invalid signature is passed to the request", async () => {
```

Comment:
Minor nitpick but it would be nice if we had some whitespace between the `it` statements